src/incidencematrix.jl

```diff
@@ -88,3 +123,35 @@ function Base.resize!(M::IncidenceMatrix{Symmetric}, n::Base.Integer)
     n >= 0 || throw(DomainError(n, "can not resize to a negative length"))
     _resize!(M,Int64(n),Int64(n))
 end
+
+function Base.show(io::IO, ::MIME"text/plain", M::IncidenceMatrix)
+    m,n = size(M)
+    print(io, "$m×$n IncidenceMatrix\n")
+    for i in 1:min(20, size(M, 1))
+        print(io, "($i) - ")
+        join(io, [i for i in M[i,:].s][1:min(20,length(M[i,:].s))], ", ")
```

Comment:
I am not really convinced of this output format. The `-` can be confused for a sign, so maybe the `(row-index) - ` could be changed to `[row-index] = ` similar to julias sparsevector. Or we can just omit the row-index since all the rows will be present anyway?

Maybe we should print it like SparseMatrixCSC containing `1` (or `true` but this makes for an ugly layout) for the existing entries and `.` otherwise?
```
3×3 SparseMatrixCSC{Int64, Int64} with 2 stored entries:
 ⋅  ⋅  1
 ⋅  2  ⋅
 ⋅  ⋅  ⋅
```

I am also open for other suggestions.

Reply:
my personal preference was to utilize the special property of an `IncidenceMatrix` that we can conclude every non-zero entry to be `true`, so we just need to state the locations of these entries. this also is close to the `polymake` design. also, i would say the information is easier to to read from this output, because we can instantly determine the indices from reading the print. that is also the reason i included that `row-index` in each line.

of course, the `SparseMatrixCSC` print creates more of a `Matrix` feeling (in that case i would also prefer `1` instead of `true`, julia does so, too, for `BitArray`), but there is no emphasis on the `Incidence`.

Reply:
I guess the standard way would be to hook into `SparseMatrixCSC` printing. If we want to see indices directly that's fine, but then it'd be better to use `displaysize(io)` than hardcoding `20` or some other constant.

Reply:
> I am not really convinced of this output format. 

That's really difficult.  I think the standard polymake output, like one set of indices for row (= facet) is the most intuitive and most compact way of printing.  For OSCAR that should be shifted by one, of course.

The data structure, by design, does not prefer rows over columns; in this sense a matrix (like that Julia type you proposed) is more adequate.  Occasionally it is useful to see this matrix (hence the polymake formatting filter `dense`), but most of the time it is too bulky.